nipap/nipap-passwd.man.rst

```diff
@@ -29,6 +34,10 @@ Options
 	delete
     	Delete user with username specifies with **-u** from the database
 
+    modify
```

Comment:
There seems to be a mixture of tabs & space for indentation here. I guess space is what we want and thus the added entry is correct, but perhaps you could fix the other entries in the list of arguments while we're at it..?